### PR TITLE
Describe template basics

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: includes/custom-templates.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+Templates are options that end with **...Template** (in jQuery, Angular, and Vue) or **...Render**/**...Component** (in React).
+
+Each template provides access to the following parameters:
```
Comment (IlyaKhD) [inline includes/custom-templates.md]:
```suggestion
Each template has access to the following parameters:
```

File: includes/custom-templates.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+- `data`     
+A data source object or an object with fields useful for this particular template. For information on the contents of `data`, refer the template's API reference article and the **Template Data** section in it.
```
Comment (IlyaKhD) [inline includes/custom-templates.md]:
```suggestion
A data source object or an object with fields specific for a particular template. For information on the contents of `data`, refer the template's API reference article and the **Template Data** section in it.
```

File: includes/custom-templates.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+                </template>
+            </DxItem>
+            <DxItem>
+                Item with a nested component
+                <template #default>
+                    <div>
+                        <DxButton text="Click me" />
+                    </div>
+                </template>
+            </DxItem>
```
Comment (IlyaKhD) [inline includes/custom-templates.md]:
```suggestion
            <DxItem>
                <template #default>               
                    <div>
                        Item with a nested component
                        <DxButton text="Click me" />
                    </div>
                </template>
            </DxItem>
```

File: includes/custom-templates.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+        </dxi-item>
+        <dxi-item>
+            <b>Item 2</b>
+        </dxi-item>
+        <dxi-item>
+            Item with a nested component
+            <div *dxTemplate>
+                <dx-button text="Click me"></dx-button>
+            </div>
+        </dxi-item>
```
Comment (IlyaKhD) [inline includes/custom-templates.md]:
```suggestion
        <dxi-item>
            <div *dxTemplate>
                Item with a nested component
                <dx-button text="Click me"></dx-button>
            </div>
        </dxi-item>
```

File: includes/custom-templates.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
+Templates are options that end with **...Template** (in jQuery, Angular, and Vue) or **...Render**/**...Component** (in React).
```
Comment (IlyaKhD) [inline includes/custom-templates.md]:
```suggestion
Templates are passed as options that end with **...Template** (in jQuery, Angular, and Vue) or **...Render**/**...Component** (in React).
```

File: includes/custom-templates.md [hunk 6]
```diff
@@ -0,0 +1,297 @@
+Templates are passed as options that end with **...Template** (in jQuery, Angular, and Vue) or **...Render**/**...Component** (in React).
+
+Each template has access to the following parameters:
+
+- `data`     
+A data source object or an object with fields specific for a particular template. For information on the contents of `data`, refer the template's API reference article and the **Template Data** section in it.
```
Comment (albertov05) [inline includes/custom-templates.md]:
```suggestion
A data source object or an object with fields specific to a particular template. For information on the contents of `data`, refer to the **Template Data** section of the template's API reference article.
```

File: includes/custom-templates.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+                    itemRender={renderListItem}
+                />
+            );
+        }
+    }
+    export default App;
+
+---
+
+Collection widgets also support templates for individual items. Do not specify the widget's **dataSource** option if you use individual templates.
```
Comment (albertov05) [inline includes/custom-templates.md:152]: I think "option if you use individual templates" should also be in italics.

**Pull request merged**